.github/workflows/broken-links-check.yml

```diff
@@ -0,0 +1,183 @@
+name: Broken link check (va.gov)
+
+on:
+  pull_request:
```

Comment:
`pull_request:` will be removed right before merge. Right now it's the only way to trigger the workflow.